source/404.md

```diff
@@ -6,13 +6,16 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed. Use menu above to navigate.
+The page was moved or never existed.  
+Use menu above to navigate or go to [home page](/).  
+If you noticed any site problem, typos, etc on any page, 
+please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
```

Comment:
Put link in `[]` in other case it won't clickable

Reply:
`[home page](/)` is  [home page](/)

Reply:
And it is clickable. (i.e. markdown is correct)

Reply:
`<https://github.com/apache/struts-site>` is <https://github.com/apache/struts-site>

clickable too.

Reply:
ha! didn't know that :) Thanks :)